Code/Framework/AzToolsFramework/Tests/ManipulatorViewTests.cpp

```diff
@@ -96,9 +94,57 @@ namespace UnitTest
         cameraState.m_position = AZ::Vector3::CreateAxisY(20.0f);
         cameraState.m_forward = -AZ::Vector3::CreateAxisY();
 
-        const float scale =
-            AzToolsFramework::CalculateScreenToWorldMultiplier(AZ::Vector3::CreateAxisX(-10.0f), cameraState);
+        const float scale = AzToolsFramework::CalculateScreenToWorldMultiplier(AZ::Vector3::CreateAxisX(-10.0f), cameraState);
 
         EXPECT_NEAR(scale, 2.0f, std::numeric_limits<float>::epsilon());
     }
+
+    TEST_F(ManipulatorViewTest, ManipulatorViewQuadDrawsAtCorrectPositionWhenManipulatorSpaceIsScaledUniformlyAndNonUniformly)
+    {
+        // Given
+        // simulate a custom manipulator space (e.g. entity transform) and a local offset within that space (e.g. spline vertex position)
+        const AZ::Transform space =
+            AZ::Transform::CreateTranslation(AZ::Vector3(2.0f, -3.0f, -4.0f)) * AZ::Transform::CreateUniformScale(2.0f);
+        const AZ::Vector3 localPosition = AZ::Vector3(2.0f, -2.0f, 0.0f);
+        const AZ::Vector3 nonUniformScale = AZ::Vector3(2.0f, 3.0f, 4.0f);
+        const AZ::Transform combinedTransform =
+            AzToolsFramework::ApplySpace(AZ::Transform::CreateTranslation(localPosition), space, nonUniformScale);
```

Comment:
might be good in future to add a test for ApplySpace as well

Reply:
Yeah true! Do you have any thoughts on where might be a good home for that (in terms of where the test(s) could live?). I'll have a think to see if anywhere might be appropriate (or we add a new file?)

Reply:
Might defer for another PR but if you have any thoughts on where this could go (maybe somewhere related to NUS?) let me know. Thanks!

Reply:
Yeah, my suggestion was for a future PR. I'll have a think as well where would make sense for it to live.